tests/test_tables.py

```diff
@@ -29,8 +29,13 @@
 
 
 @pytest.fixture
-def table_data_dict(table_data: List[str]) -> Dict[EpicsName, RecordValue]:
-    return {EpicsName(EPICS_FORMAT_TABLE_NAME): table_data}
+def table_data_1_dict(table_data_1: List[str]) -> Dict[EpicsName, RecordValue]:
+    return {EpicsName(EPICS_FORMAT_TABLE_NAME): table_data_1}
+
+
+@pytest.fixture
+def table_data_2_dict(table_data_2: List[str]) -> Dict[EpicsName, RecordValue]:
+    return {EpicsName(EPICS_FORMAT_TABLE_NAME): table_data_2}
```

Comment:
It doesn't look like this is ever used? Is there a test missing that should use it?

Reply:
I don't think so, there's a table test that needs the `table_data_1_dict`, I probably accidentaly made a second for the other table data